apps/web/src/hooks/useStablecoinPrice.ts

```diff
@@ -41,7 +41,7 @@ export const STABLECOIN_AMOUNT_OUT: { [key in SupportedInterfaceChain]: Currency
   [ChainId.ARBITRUM_GOERLI]: CurrencyAmount.fromRawAmount(USDC_ARBITRUM_GOERLI, 10_000e6),
   [ChainId.POLYGON_MUMBAI]: CurrencyAmount.fromRawAmount(USDC_POLYGON_MUMBAI, 10_000e6),
   [ChainId.CELO_ALFAJORES]: CurrencyAmount.fromRawAmount(CUSD_CELO_ALFAJORES, 10_000e6),
-  [ChainId.HEMI_SEPOLIA]: CurrencyAmount.fromRawAmount(TESNET_DAI_HEMI_SEPOLIA, 10_000e18),
+  [ChainId.HEMI_SEPOLIA]: CurrencyAmount.fromRawAmount(DAI_HEMI_SEPOLIA, 10_000e18),
```

Comment:
I think this should use USDC, but I see you're not adding it as part of this PR so... just commenting it for future reference 😄

Reply:
Not sure if it should use USDC, some chains use USDC, others USDT or DAI.

Reply:
well, optimism uses dai, so let's use that 🤷🏽